### PR TITLE
feat: Responder and Requester sample apps for Auth SDK

### PR DESCRIPTION
**Pull request by Elyniss**

### CODE REVIEW

File: authSDK/requester/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    kotlinOptions {
+        jvmTarget = jvmVersion.toString()
+    }
+    buildFeatures {
+        viewBinding = true
+    }
+}
+
+dependencies {
+    implementation(project(":auth"))
```
Comment (TalhaAli00) [inline authSDK/requester/build.gradle.kts:39]: let's move the sample_common to this project

Comment (Elyniss) [reply authSDK/requester/build.gradle.kts:39]: Added dependency on samples_common module. Also created a task to extract it from sign SDK #357

File: authSDK/requester/src/main/kotlin/com/walletconnect/requester/RequesterApplication.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.walletconnect.requester
+
+import android.app.Application
+
+class RequesterApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        // todo: Do some WC AuthSDK Init magic here
```
Comment (jakobuid) [inline authSDK/requester/src/main/kotlin/com/walletconnect/requester/RequesterApplication.kt:8]: :D

File: authSDK/requester/src/main/kotlin/com/walletconnect/requester/ui/connect/pairing_generate/PairingGenerationDialogFragment.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    private val binding by viewBinding(DialogConnectUriBinding::bind)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        dialog?.window?.setLayout(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+
+        //todo: Reimplement. Right now only for demo purposes
+        viewModel.connectToWallet {
+            val pairingUri = "https://walletconnect.com/" // todo: how to get right uri here?
```
Comment (jakobuid) [inline authSDK/requester/src/main/kotlin/com/walletconnect/requester/ui/connect/pairing_generate/PairingGenerationDialogFragment.kt:30]: From AuthClient.request method

File: authSDK/responder/src/main/res/navigation/nav_graph.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    </navigation>
+
+    <fragment
+        android:id="@+id/fragment_sessions"
+        android:name="com.walletconnect.responder.ui.sessions.SessionsFragment"
+        android:label="Sessions"
+        tools:layout="@layout/fragment_sessions" />
+
+    <!--    <action-->
```
Comment (jakobuid) [inline authSDK/responder/src/main/res/navigation/nav_graph.xml:58]: Let's remove if not used

**Review by jakobuid — APPROVED** [review]

**Review by TalhaAli00 — APPROVED** [review]

**Pull request merged**